WordPress/Classes/Networking/BlogServiceRemoteREST.m

```diff
@@ -19,6 +19,27 @@ - (id)initWithApi:(WordPressComApi *)api
     return self;
 }
 
+- (void)checkMultiAuthorForBlog:(Blog *)blog success:(void(^)(BOOL isMultiAuthor))success failure:(void (^)(NSError *error))failure
```

Comment:
Break a bit so this line doesn't exceed 80 chars.

Also use the typedef'd block names for the parameters.